### DISCUSSION TITLE
下载合集(播放列表）视频最高画质不一致时 无法选择优先下载最高画质

### DISCUSSION THREAD

**Discussion by TPAXcc** [ywmoyue/biliuwp-lite#1097] [General]
### Bug 描述

下载合集(播放列表）视频最高画质不一致时 无法选择优先下载最高画质
如有的视频 720p 有的视频 1080p
则下载时最高画质只能选 720p

另：分P视频无此bug

### 操作步骤

```plain text
https://b23.tv/BV1Y64y1B7QC
```

### 期望结果

。

### 应用来源

Github/其他Github镜像站

### 代码贡献

我更擅长发现问题

**Comment by ProJend** [comment]
@ywmoyue 这个问题是不是在 #1064 处理过了

**Comment by ywmoyue** [comment]
@ProJend 不是同一个问题，#1064 处理的是当前视频的下载选项不正确问题

**Comment by ProJend** [comment]
> [@ProJend](https://github.com/ProJend) 不是同一个问题，[#1064](https://github.com/ywmoyue/biliuwp-lite/issues/1064) 处理的是当前视频的下载选项不正确问题

效果有了
![Image](https://github.com/user-attachments/assets/98ba86ba-9030-4921-8fd2-c782d27f93e3)

**Comment by ywmoyue** [comment]
@ProJend  我想 @TPAXcc 的意思是合集中其他视频的画质更高时无法选择对应的画质

不过其实这个问题没法处理，不可能去把合集中全部视频的画质选项请求下来，短时间的大量请求很可能会被b风控

我先转discussion吧，后面如果发现合集数据带有画质选项的话再开issue
